### PR TITLE
Reformat the control-plane-node-communication page

### PR DESCRIPTION
**Pull request by tengqm**
This PR does two things:

- Change 'apiserver' to 'API server' according to the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-a-general-descriptor-over-a-component-name).
- Wrap the long lines appropriately for ease of change tracking.

### CODE REVIEW

**Review by sftim — CHANGES_REQUESTED** [review]
One thing, just slightly more than a nit.

File: content/en/docs/concepts/architecture/control-plane-node-communication.md
Comment (sftim) [inline content/en/docs/concepts/architecture/control-plane-node-communication.md:14]: Since we're touching this text, we should avoid implying that “control plane” == `kube-apiserver`. Instead, what we'd say is that the API server is commonly the only part of the control plane that communicates with components outside the control plane (the _hub and spoke_ API pattern).

File: content/en/docs/concepts/architecture/control-plane-node-communication.md
Comment (sftim) [inline content/en/docs/concepts/architecture/control-plane-node-communication.md:23]:
```suggestion
Kubernetes has a _hub and spoke_ API pattern. All API usage from nodes (or the pods they run)
```

File: content/en/docs/concepts/architecture/control-plane-node-communication.md
Comment (sftim) [inline content/en/docs/concepts/architecture/control-plane-node-communication.md:63]:
```suggestion
* Attaching (usually through `kubectl`) to running pods.
```
?

**Review by sftim — COMMENTED** [review]
Does this PR need an update in light of the SOCKS changes from #33132? 

Also, some small tweaks.

**Comment by sftim** [comment]
My quibble from https://github.com/kubernetes/website/pull/33132#pullrequestreview-959644879 stands. As to the SOCKS thing, we might address that by opening and freezing an issue, then accepting this PR without the SOCKS updates.

**Comment by tengqm** [comment]
@sftim I don't know what you are talking about when referring to #33132, which is the PR number of this one.

**Review by sftim — APPROVED** [review]
Happy to approve this:
- the intent makes sense to me
- I'll let a review provide LGTM if they are happy
- I'm happy that @tengqm will make any further commits (if needed) only in line with the existing changes here

**Comment by sftim** [comment]
The quibble was https://github.com/kubernetes/website/pull/33132#pullrequestreview-959644879 (and that quibble remains, in my view)

_(lightly reworded)_
> The previous title (“Control Plane-Node Communication”) didn't imply a single direction of communication and the new wording does. Can we find a different rewording?

(the new title implies that the control plane sends messages to the node, whereas in fact the page also covers communications that nodes initiate back to the control plane)

**Comment by sftim** [comment]
@tengqm, with the title tweaked this is a really nice improvement. Thanks!

**Pull request merged**
